### PR TITLE
Update npp.txt

### PR DESCRIPTION
**Pull request by minecraftchest1**
Updated download link and checksum.

### CODE REVIEW

**Comment by learn-more** [comment]
Did you test this on ReactOS ?
Please provide a screenshot.

**Comment by minecraftchest1** [comment]
image
Does throw an error on the first launch after full install, but works as far as I can tell.

**Comment by minecraftchest1** [comment]
Forgive me, but is this going to be merged?

**Comment by minecraftchest1** [comment]
Moved to https://github.com/reactos/rapps-db/pull/101

**Comment by minecraftchest1** [comment]
Latest commit contains multiple files. Just a headsup.

File: npp.txt [hunk 1]
```diff
@@ -3,13 +3,13 @@ Name = Notepad++
 Version = 6.9
 License = GPL
 Description = A full featured, free text editor with many plug-ins.
-Size = 4.0 MiB
+Size = 3684 B
```
Comment (learn-more) [inline npp.txt:6]: Just delete this line

**Comment by minecraftchest1** [comment]
So remove the Size line?

**Comment by learn-more** [comment]
> So remove the Size line?

yes.
If the SizeInBytes is present, the Size is not needed.

**Comment by minecraftchest1** [comment]
Thanks. I'll do so.

**Comment by minecraftchest1** [comment]
Done yesterday.

File: npp.txt [hunk 2]
```diff
@@ -3,13 +3,12 @@ Name = Notepad++
...
-Size = 4.0 MiB
 Category = 6
 URLSite = http://notepad-plus-plus.org/
-URLDownload = https://notepad-plus-plus.org/repository/6.x/6.9/npp.6.9.Installer.exe
-SHA1 = 99d3507a8eab110b7471bf2a8e63e88ecfb4e831
+URLDownload = https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v7.8.7/npp.7.8.7.Installer.exe
+SHA1 = 9BDD1D7C3C26063B30A9C3719BFA54B8BB7CD628
 CDPath = none
-SizeBytes = 4206592
+SizeBytes = 3684
```
Comment (learn-more) [inline npp.txt:11]: It went down from 4 MB to 3.6 kb?
I find that hard to believe.

**Comment by minecraftchest1** [comment]
By what i can see, it downloads the installer. I'm sure if I did the self-contained portable it would be larger.

File: npp.txt (hunk 2 above)
Comment (binarymaster) [inline npp.txt:11]: This is **size in bytes**, not in kilobytes @minecraftchest1 
```suggestion
SizeBytes = 3772184
```

**Comment by minecraftchest1** [comment]
How did you get the size? Unless I misunderstood the output of the "du" command on linux, I believe what I put is correct.

And I don't think I ignored anyone's comments, unless there was a recent discussion on mattermost I missed.

**Comment by binarymaster** [comment]
image

**Comment by minecraftchest1** [comment]
I stand corrected.

**Comment by minecraftchest1** [comment]
Changes pushed.

File: npp.txt [hunk 3]
```diff
@@ -3,13 +3,12 @@ Name = Notepad++
 Version = 6.9
 License = GPL
 Description = A full featured, free text editor with many plug-ins.
-Size = 4.0 MiB
 Category = 6
 URLSite = http://notepad-plus-plus.org/
-URLDownload = https://notepad-plus-plus.org/repository/6.x/6.9/npp.6.9.Installer.exe
-SHA1 = 99d3507a8eab110b7471bf2a8e63e88ecfb4e831
+URLDownload = https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v7.8.7/npp.7.8.7.Installer.exe
+SHA1 = 9BDD1D7C3C26063B30A9C3719BFA54B8BB7CD628
```
Comment (binarymaster) [inline npp.txt:9]: SHA1 hash should be lowercase.

File: npp.txt [hunk 4]
```diff
@@ -55,4 +54,3 @@ Description = 一种全功能，带有很多插件的免费文本编辑器。
 
 [Section.0813]
 Description = Een teksteditor.
-
```
Comment (binarymaster) [inline npp.txt:58]: Unnecessary change.

**Comment by minecraftchest1** [comment]
Changes hash to lowercase.

**Comment by JoachimHenze** [comment]
We intentionally downgraded NPP to older v6 versions, because NPP v7+ adds a dependency of its updater GUP.exe to libcurl.dll, which is compiled for Vista+ (actually without any need. They could simply fix it.)
See also d5edb8f123542ea3598ef958a574458aad62a31f and 52a8791ef248407d9fdeb6231268fd87a4d831c1
Therefore I close the PR.
If you want newer NPP versions to be added to rapps, it makes sense to flood the NPP bugtracker instead:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4603

**Comment by minecraftchest1** [comment]
I didn't have any issues installing it. But I will probably wan't to provide a working download for version 6.
